components/bl-audio-player-component/src/player-components/track-time.js

```diff
@@ -16,8 +16,12 @@ export function TrackTime({ audioRef, timer }) {
   return (
     <div className="track-time">
       <div>{ timer }</div>
-      <span>/</span>
-      <div>{ trackLength }</div>
+      { trackLength && (
+        <span>/</span>
+      ) }
+      { trackLength && (
+        <div>{ trackLength }</div>
+      ) }
```

Comment:
```
      { trackLength && (
      <>
        <span>/</span>
        <div>{ trackLength }</div>
      </>
      ) }
```

or 

```
      { trackLength && (<span>/ { trackLength }</span>) }
```
      
I prefer second if there are no need in div

Reply:
I chose the first variant because the space between the elements is controlled by the span margin.